.licenserc.yaml

```diff
@@ -0,0 +1,98 @@
+header:
+  - paths:
+      - "**/*.go"
+    paths-ignore:
+      - "**/agent/**"
+      - "**/admin/**"
+      - "**/managed/**"
+      - "**/api-tests/**"
+      - "**/mock_*"
+      - "**/*.pb.go"
+      - "**/*_reform.go"
+      - "**/*.pb.gw.go"
+      - "**/json/client/**"
+      - "**/amclient/**"
+      - "**/ammodels/**"
+    license:
+      spdx-id: AGPL-3.0
+      content: |
+        // pmm
+        // Copyright (C) 2019 Percona LLC
+        //
+        // This program is free software: you can redistribute it and/or modify
+        // it under the terms of the GNU Affero General Public License as published by
+        // the Free Software Foundation, either version 3 of the License, or
+        // (at your option) any later version.
+        //
+        // This program is distributed in the hope that it will be useful,
+        // but WITHOUT ANY WARRANTY; without even the implied warranty of
+        // MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+        // GNU Affero General Public License for more details.
+        //
+        // You should have received a copy of the GNU Affero General Public License
+        // along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+  - paths:
+      - "**/managed/**/*.go"
+      - "**/api-tests/**/*.go"
+    paths-ignore:
+      - "**/*_reform.go"
+      - "**/mock_*"
+    license:
+      spdx-id: AGPL-3.0
+      content: |
+        // pmm-managed
```

Comment:
The only difference between this section and previous is this line. I guess we can merge them and use `Percona Monitoring and Management` in all headers, or omit this part at all.

@atymchuk what do you think?

Reply:
I prefer omitting that line.

Reply:
@idoqo let's remove this first line from all headers and simplify config for license checker

Reply:
yeah, I'll simplify the license checker config file - it works fine as long as files have the license text in them - so I'll set up a script to remove the license header from the source files later and make that a separate PR.